intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+---
+layout: default
+title: Basic window setup
+---
+
+<body>
+    <h1>Basic window setup for pygame</h1>
+    <p>In this article, we will go over the structure to open and keep the window running in pygame.</p>
+
+    <h2>Table of contents</h2>
+    <p>
+        <ol id="toc">
+            <!--
+            To be filled by javascript
+            -->
+        </ol>
+    </p>
+
+    <h2><a id="what-is-pygame" class="js"></a>What is pygame?</h2>
+    <p>
+        Before we get into the nitty gritty parts of it, let us first understand what pygame is.
+        pygame is a free and open source library for the Python programming language, which allows 
+        us to build multimedia applications such as video games.
+    
+    </p>
+    <p>
+        pygame provides the very basic utility to draw and manipulate pixels 
+        on to the screen, much like SDL2, but with extended utility in various forms 
+        that makes it easier to use. pygame gives you direct control over how you 
+        want the code to run, this means you could also easily do things that could 
+        drastically drop performance if you aren't careful. This article guides you 
+        through how to avoid such things, while also giving you deeper 
+        understanding of pygame and how to use it.         
+    </p>
+
+    <h2><a id="prerequisites" class="js"></a>Prerequisites</h2>
+    <p>
+        As pygame is a library for <i>python</i>, it is necessary that you 
+        are familiar with Python itself. This goes beyond the scope of this book, but 
+        there are some other great resources for learning Python, such  
+        as <a href="">automate the boring stuff</a>.
+        
+        Now, luckily you don't have to be an Anthony Shaw or Tim Peters to be
+        able to use pygame, just some basic Python will do. 
+        This includes:
+            <ul>
+                <li>Understanding how types and instances work</li>
+                <li>Knowing how to access and modify attributes of objects</li>
+                <li>Familiarity with basic OOP(Object Oriented Programming)</li>
+                <li>Basic control flow keywords such as - for, while, break, if, elif, else</li>
+                <li>Functions</li>
+            </ul>
+        
+        Since in Python everything is an object, that is also how you would 
```

Comment:
You're slightly inconsistent with your capitalization of "Python". Here you have it capitalized, but on line 38 you have it lowercased.